test/controllers/TransferSummaryControllerSpec.scala

```diff
@@ -68,8 +68,9 @@ class TransferSummaryControllerSpec extends FrontEndTestHelper {
       val seriesCode = Some(gcs.GetConsignment.Series(Some("Mock Series")))
       val transferringBodyName = Some(gcs.GetConsignment.TransferringBody(Some("MockBody")))
       val totalFiles: Int = 3
+      val consignmentReference = Option("TEST-TDR-2021-MTB")
```

Comment:
Probably should use `Some` constructor instead of `Option` constructor, as sure the value is not a `null`

Reply:
Cool, I'll change it to `Some` 👍